### ISSUE TITLE
Camera hangs when redirecting to the application

### ISSUE THREAD

**Issue by greg-devtac** [apache/cordova-plugin-camera#352]
When im capturing the image and accept the image it hangs the application. Once you terminate the application and caputre again and accept the image it is succesfully redirected to the application and the application doesnt hang.

**Comment by jcesarmobile** [comment]
platform? plugin version? OS version of the device?

**Comment by greg-devtac** [comment]
hi @jcesarmobile 
 here's the details 

OS : Lollipop 5.1
Platform: Android
Plugin version : 4.9.0

**Comment by greg-devtac** [comment]
Sorry @jcesarmobile yes you are correct. 

the 4.9.0 it was the 
`"@ionic-native/camera": "4.9.0"`

**Comment by janpio** [comment]
Can you please create a minimal reproduction app with `cordova create` and put it on Github @greg-devtac? Thanks.

**Comment by jcesarmobile** [comment]
I couldn't reproduce on an Ionic app yesterday, so going to close this.

If you still face the issue, you can create a new one but providing a sample project where it can be easily reproduced.

**Comment by yuankunluo** [comment]
same problem here with OnePlus 6T , android 9.0.1.
Ionic 4 rc.01.
The problem was caused by camera plugin, which is not stable under Android 9.

The OS kill the system and restart the App. There is no "create" life event hock in @ionic/angular.

The app was killed not pushed into the background. So we can not get anything from "resume" life event.

I have tested on 3 Phones. It occues only on Android 9, it seems not stable and unpredictable. Not happens always. 
My other two phones, Huawei p20 and Mi 3 both running on Android 8. There are no problem on them.
